docs/system/system-metrics.md

```diff
@@ -11,6 +11,12 @@ metrics in OpenTelemetry. Consider the [general metric semantic
 conventions](/docs/general/metrics.md#general-metric-semantic-conventions) when creating
 instruments not explicitly defined in the specification.
 
+The `system.*` namespace SHOULD be exclusively used to report hosts' metrics.
+The `system.*` namespace SHOULD only be used when the metrics are collected from within the target system. (physical servers, virtual machines etc).
+Metrics collected from technology-specific, well-defined APIs (e.g. Kubelet's API or container runtimes)
+should be reported under their respective namespace (e.g. k8s.*, container.*).
+For resource attributes related to a host the `host.*` namespace should be used.
```

Comment:
```suggestion
For resource attributes related to a host, the `host.*` namespace SHOULD be used.
```
to be consistent with SHOULD as above, OR

```suggestion
For resource attributes related to a host, use the `host.*` namespace.
````

To not have a should in the sentence.